spec/lib/newspaper_works/data/work_derivative_loader_spec.rb

```diff
@@ -4,6 +4,46 @@
 RSpec.describe NewspaperWorks::Data::WorkDerivativeLoader do
   include_context "shared setup"
 
+  describe "enumerates available derivatives" do
+    it "includes expected derivative path for work" do
+      work = sample_work
```

Comment:
Is there a way to DRY these specs up a bit? Not sure if this works, but something like this at the beginning:
```ruby
let (:work) { sample_work }
before {
  mk_txt_derivative(work)
  work.save!(validate: false)
}
```

Reply:
Yeah, that's doable.  `sample_work` is in a shared context let statement (in `spec/misc_shared.rb`), so either I just replace `work` with `sample_work` or just have a single shorthand `let` like above for this file.